docs/config.ld

```diff
@@ -109,9 +116,6 @@ file = {
 
         -- Ignore some parts of the widget library
         '../lib/awful/widget/init.lua',
-
-        -- Ignore the drawing window, users should only care about wiboxes
-        '../objects/drawin.c',
```

Comment:
Why? What was wrong with this? Also, I'd preserve the comment. You could e.g. do
```
-- Ignore the ....
-- '../objects/drawin.c',
```

Reply:
I stopped using a wildcard, so it's no longer necessary

Reply:
Okay, but I'd still like to have this more explicit. Why do we have these docs in the file at all if we don't want them?

Reply:
![image](https://user-images.githubusercontent.com/340384/32987908-03843efe-ccc6-11e7-891f-5b57a4f8ef57.png)

Because ldoc produces those dummy entries if the wildcard is used.